src/drop-down-menu.jsx

```diff
@@ -65,6 +66,7 @@ let DropDownMenu = React.createClass({
 
   getStyles(){
     let zIndex = 5; // As AppBar
+    let disabled = this.props.disabled;
```

Comment:
More or less. It should be declared `const` because `disabled` is not and should probably not be mutated (yes because he come from a prop which is a source of truth).